### PR TITLE
fix issue 864

### PR DESCRIPTION
**Pull request by Thor-wl**
Signed-off-by: Thor <1187526662@qq.com>

### CODE REVIEW

File: pkg/scheduler/plugins/util/framework.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
+package util
```
Comment (k82cn) [inline pkg/scheduler/plugins/util/framework.go:1]: let's move to `pkg/scheduler/plugins/util/k8s/framework.go`

**Comment by k82cn** [comment]
LGTM overall, please help to make CI happy :)

File: pkg/scheduler/plugins/util/snapshot.go [hunk 2]
```diff
@@ -17,7 +17,7 @@ limitations under the License.
 // Copied from https://github.com/kubernetes/kubernetes/blob/v1.18.3/pkg/scheduler/internal/cache/snapshot.go
 // as internal package is not allowed to import
 
-package cache
+package util
```
Comment (k82cn) [inline pkg/scheduler/plugins/util/snapshot.go:20]: also put this into k8s.

Comment (Thor-wl) [reply pkg/scheduler/plugins/util/snapshot.go]: done

File: pkg/scheduler/plugins/util/framework.go (hunk 1 above)
Comment (Thor-wl) [reply pkg/scheduler/plugins/util/framework.go]: done

File: pkg/scheduler/plugins/util/k8s/framework.go [hunk 3]
```diff
@@ -0,0 +1,88 @@
+package k8s
```
Comment (hzxuzhonghu) [inline pkg/scheduler/plugins/util/k8s/framework.go:1]: add copyright please

Comment (Thor-wl) [reply pkg/scheduler/plugins/util/k8s/framework.go:17]: done

**Pull request merged**